src/main/java/eu/stamp_project/testrunner/runner/coverage/JUnit4JacocoRunnerCoveredResultPerTestMethod.java

```diff
@@ -25,10 +25,12 @@ public JUnit4JacocoRunnerCoveredResultPerTestMethod(String classesDirectory, Str
 
 	public JUnit4JacocoRunnerCoveredResultPerTestMethod(String classesDirectory, String testClassesDirectory, List<String> blackList, CoverageTransformer coverageTransformer) {
 		super(classesDirectory, testClassesDirectory, blackList, coverageTransformer);
+		System.out.println("I'M EVERYWHERE : " + classesDirectory);
```

Comment:
Could you remove these sout (for debugging I guess)? :feelsgood:

Reply:
Ups, forgot I had those there :grimacing: